protocols/CollectGCBiasMetrics.sh

```diff
@@ -17,7 +17,7 @@
 #string picardJar
 #string insertSizeMetrics
 #string gcBiasMetrics
-#string	project
+^#string project
```

Comment:
Why is there a ^ at the beginning of the modified lines?

Reply:
Thaaat's a good question. Looks like I messed up my regex. Maybe just burn this pull request.

Reply:
Consistent white space is nice though ;)

Reply:
Alright, just made it worse, then made it better. should be fixed now.